pkg/build/docker_go.go

```diff
@@ -371,17 +371,17 @@ ARG BUILD_TAGS
 ENV TESTPLAN_EXEC_PKG ${TESTPLAN_EXEC_PKG}
 
 # PLAN_DIR is the location containing the plan source inside the container.
-ENV PLAN_DIR /plan/
+ENV PLAN_DIR /plan
```

Comment:
No trailing slash for the default `PLAN_DIR`, which makes it more intuitive to use in the following hooks.